ru/turing/courses/lesson2/Person.java

```diff
@@ -0,0 +1,110 @@
+package ru.turing.courses.lesson2;
+
+import java.time.DateTimeException;
+import java.time.LocalDate;
+import java.time.Period;
+import java.time.format.DateTimeFormatter;
+import java.util.regex.Pattern;
+
+
+public class Person {
+    private String name, surname, secondName, address;
+    private LocalDate data;
+
+    public String getIm() {
+        return name;
+    } //геты
+
+    public String getFam() {
+        return surname;
+    }
+
+    public String getOtch() {
+        return secondName;
+    }
+
+    public void setFIO(String surname, String name, String secondName) {//ввод фио
+        this.surname = surname;
+        this.name = name;
+        this.secondName = secondName;
+    }
+
+    public boolean setData(String date, String regex) {//ввод даты и смена её формата
+        DateTimeFormatter Form = DateTimeFormatter.ofPattern(regex);//задание формата строки
+        try {
+            this.data = LocalDate.parse(date, Form);//перевод строки с форматом в дату
+            return true;
+        } catch (DateTimeException exception) {
+            return false;
+        }
+    }
+
+    public boolean setAdr(String Sadr, String regex) {//ввод адреса
+        if (Pattern.matches(regex, Sadr)) {//если строка соответсвует щаблону, то сохраняем её
+            this.address = Sadr;
+            return true;
+        } else {
+            return false;
+        }
+    }
+
+    public int getmonth() {//вывод всех прожитых месяцев
```

Comment:
```suggestion
    public int getMonth() {//вывод всех прожитых месяцев
```
Несколько разных методов могут называться одинаково, и стиль кода надо выдерживать везде)